docs/guides/settings.rst

```diff
@@ -60,6 +64,10 @@ where:
 - ``$user_guid`` Is the user you want to retrieve these for (defaults to the currently logged in user)
 - ``$plugin_name`` Is the name of the plugin (detected if run from within a plugin)
 
+.. note::
+
+   Settings stored as an array will be returned as the serialized value.  Developers are expected to unserialize such settings on retrieval.
```

Comment:
Devs shouldn't have to guess which strings are serialized data structures. JSON seems less bad for this.

Bad for security: This lets the client dictate the storage model and doesn't validate the value before serialization.